src/CMakeLists.txt

```diff
@@ -18,18 +22,30 @@ set(SOURCES
     vdm.c
 )
 
-set(CORE_DEPS
-	${CORE_SOURCE_DIR}/alloc.c
-	${CORE_SOURCE_DIR}/os_posix.c
-	${CORE_SOURCE_DIR}/os_thread_posix.c
-	${CORE_SOURCE_DIR}/out.c
-	${CORE_SOURCE_DIR}/util.c
-	${CORE_SOURCE_DIR}/util_posix.c)
+if(WIN32)
+	set(CORE_DEPS
+		${CORE_SOURCE_DIR}/alloc.c
+		${CORE_SOURCE_DIR}/os_windows.c
+		${CORE_SOURCE_DIR}/os_thread_windows.c
+		${CORE_SOURCE_DIR}/out.c
```

Comment:
out.c and util.c are not windows/linux specific if I remember correctly.

Reply:
I've already changed it, so that those files are added for both linux and windows outside of if-clauses. I'm not sure why the previous version of this code was shown on gh but it should be fixed now.